### ISSUE TITLE
Review & improve error page accessibility

### ISSUE THREAD

**Issue by bcmn** [bbc/simorgh#1348]
**Is your feature request related to a problem? Please describe.**
Coming out of [this comment](https://github.com/bbc/simorgh/pull/1295#issuecomment-466988524).

**Describe the solution you'd like**
We shouldn't be using CSS for the list bullets, as this will read out as 'bullet' on some screen readers. It’s recommended to use a list style of none in the CSS e.g. this means adding our own ‘bullet’, and in addition adding aria roles to the list and list items of ‘list’ and ‘listitem’ to ensure we have list semantics in when using VoiceOver.

A developer accessibility swarm should be carried out as part of this work.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

**Comment by jamesbhobbs** [comment]
@greenc05 another one for you to follow up on

**Comment by greenc05** [comment]
@jamesdonoh would you be the right person for me to follow up on this? Do we plan to work on this?

**Comment by sareh** [comment]
@greenc05 I'll be picking up this work as part of the Shared Components pod. Let me know if you have any specific points that are missing in the issue description.

**Comment by greenc05** [comment]
Thanks @sareh I will review the description next week - How do I preview an error page?

**Comment by sareh** [comment]
@greenc05 You can look at a 404 page locally by going to an article that doesn't exist in the fixture data, e.g. `http://localhost.bbc.com:7080/news/articles/c0000000039o`

**Comment by greenc05** [comment]
@sareh the description has been reviewed, there maybe another Issue at a later date to look at tweaking the h1

**Comment by joshcoventry** [comment]
Replaced by new parent issue, will require refinement: https://github.com/bbc/simorgh/issues/7815
